### PR TITLE
HPCC-16971 Add 3 additional options for WUInfo

### PR DESCRIPTION
**Pull request by wangkx**
Add options for not including ECL/Helpers/AllowedClusters in WUInfo

Signed-off-by: wangkx <kevin.wang@lexisnexis.com>

### CODE REVIEW

**Comment by wangkx** [comment]
@afishbeck please review.

File: esp/services/ws_workunits/ws_workunitsHelpers.cpp [hunk 1]
```diff
@@ -487,20 +487,25 @@ void WsWuInfo::getHelpers(IEspECLWorkunit &info, unsigned flags)
...
         else
         {
-            SCMStringBuffer qname;
-            query->getQueryShortText(qname);
-            if(qname.length())
+            if (flags & WUINFO_IncludeECL)
             {
-                if((flags & WUINFO_TruncateEclTo64k) && (qname.length() > 64000))
-                    qname.setLen(qname.str(), 64000);
+                SCMStringBuffer qname;
```
Comment (afishbeck) [inline esp/services/ws_workunits/ws_workunitsHelpers.cpp]: Strange name for the variable to store ECL text.

**Comment by afishbeck** [comment]
@wangkx other than picky comment, looks good.

File: esp/services/ws_workunits/ws_workunitsHelpers.cpp (hunk 1 above)
Comment (wangkx) [inline esp/services/ws_workunits/ws_workunitsHelpers.cpp]: It comes from old code. But, I will rename it to 'queryText'..

**Comment by wangkx** [comment]
@afishbeck please review my changes.

**Comment by afishbeck** [comment]
@wangkx Much better!

**Pull request merged**